src/Compilers/CSharp/Portable/Binder/ExtensionMethodScope.cs

```diff
@@ -70,17 +68,8 @@ public bool MoveNext()
             else
             {
                 var binder = _current.Binder;
-                if (!_current.SearchUsingsNotNamespace)
-                {
-                    // Return a scope for the same Binder that was previously exposed
-                    // for the namespace, this time exposed for the usings.
-                    _current = new ExtensionMethodScope(binder, searchUsingsNotNamespace: true);
-                }
-                else
-                {
-                    // Return a scope for the next Binder that supports extension methods.
-                    _current = GetNextScope(binder.Next);
-                }
+                // Return a scope for the next Binder that supports extension methods.
+                _current = GetNextScope(binder.Next);
```

Comment:
Nit: it looks like this whole if/else could be simplified to `_current = GetNextScope(_current.Binder?.Next ?? _binder);`